packages/toolpad-core/package.json

```diff
@@ -39,7 +40,8 @@
   "scripts": {
     "clean": "rimraf build",
     "prebuild": "pnpm clean",
-    "build": "pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files",
+    "build": "pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files && pnpm esmify",
+    "esmify": "rm -rf ./build/node && rm -rf ./build/*/package.json",
```

Comment:
Next.js resolver gets very confused about these `@toolpad/core/*/package.json` files. It loads CJS from `pages/_app.tsx` and ESM from `pages/auth/signin.tsx`. Forcing the package to be ESM-only solves all issues of double contexts being created.

Will adapt the build script in core with a mode for esm only.